cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -199,8 +199,8 @@ getPrunes :: SyncEnv -> Bool
 getPrunes = do
   DB.pcmPruneTxOut . getPruneConsume
 
-fullInsertOptions :: Bool -> InsertOptions
-fullInsertOptions useLedger = InsertOptions True useLedger True True True True True True True
+fullInsertOptions :: Bool -> Strict.Maybe [Word64] -> InsertOptions
+fullInsertOptions useLedger maybeKeepMNames = InsertOptions True useLedger True True True True maybeKeepMNames True True True
```

Comment:
Since it's `full` there is no need to pass `maybeKeepMNames` here. It can be hardcoded to keep all metadata.

Reply:
yeah sorry I miss read @sgillespie review comment 😅